scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -1609,6 +1599,12 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
     }))(location)
   }
 
+  private def isEolAfterColonFewerBracesBody(): Boolean = peekToken match {
```

Comment:
We don't check for colon at all here, so this is really just `isNextIndentation`, no?